.github/workflows/main.yml

```diff
@@ -6,9 +6,9 @@ name: CI
 on:
   # Triggers the workflow on push or pull request events but only for the main branch
   push:
-    branches: [ main ]
+    branches: [ main, 'week**', 'test**']
```

Comment:
why do we need to run the action on push?

Reply:
That was leftover from the template I used. We'll go ahead and only run this from PR.